src/client/components/input/drag-and-drop.tsx

```diff
@@ -111,7 +115,18 @@ function DragAndDrop({name}: Props): JSX.Element {
 
 DragAndDrop.displayName = 'DragAndDrop';
 DragAndDrop.propTypes = {
+	initialAchievement: PropTypes.shape({
+		badgeUrl: PropTypes.string,
+		id: PropTypes.number,
```

Comment:
Here we don't need the `| null` like above, as a previously selected achievement will always have an id.